EIPS/eip-1328.md

```diff
@@ -0,0 +1,49 @@
+---
+eip: 1328
+title: WalletConnect Standard URI Format
+authors: ligi <ligi@ligi.de>, Pedro Gomes <pedrouid@protonmail.com>
+type: Standards Track
+category: ERC
+status: WIP
```

Comment:
This should be `draft`. If you don't want this PR merged, put `[WIP]` in the PR title.